components/CookieConsentConfig.js

```diff
@@ -0,0 +1,133 @@
+/**
```

Comment:
This file shouldn't be in the components folder. Move `pluginConfig` to the component it is used in.

Reply:
Done ✅